CODEOWNERS

```diff
@@ -318,6 +318,7 @@
 /include/bluetooth/adv_prov.h             @nrfconnect/ncs-si-bluebagel
 /include/bluetooth/mesh/                  @nrfconnect/ncs-paladin
 /include/bluetooth/services/fast_pair/    @nrfconnect/ncs-si-bluebagel
+/include/bluetooth/services/hids.h        @nrfconnect/ncs-si-bluebagel
```

Comment:
btw. Should we apply the same for `/include/bluetooth/services/hogp.h` + sources ? (related implementation of HIDS Client)